code/datums/flock/flocktilegroup.dm

```diff
@@ -24,7 +22,7 @@
 	..() //linter machine go ANGRY
 
 /datum/flock_tile_group/proc/process()
```

Comment:
If it just calls the parent then it doesn't need to exist

Reply:
More specifically, it doesn't need to be re-defined for the subtype.

Reply:
weirdly it seems to need to exist, there's a call on all datums:process() deep in the code, and I guess /datum doesn't implement it. So it isn't an override, and should just return 0, not call the parent.

Reply:
I hate it

Reply:
me too. We can probably remove this entire thing if we're ditching the power stuff tbh.

I'm hesitant to remove it unless we're sure though.